core/src/main/java/org/jboss/pnc/rex/rest/InternalEndpointImpl.java

```diff
@@ -50,7 +50,7 @@ public InternalEndpointImpl(TaskProvider provider, OptionsProvider optionsProvid
     @Override
     @Retry
     @Fallback(fallbackMethod = "fallback", applyOn = {RollbackException.class, ArcUndeclaredThrowableException.class})
-    @RolesAllowed("user")
+    @RolesAllowed({ "pnc-users-rex-admin", "pnc-users-admin" })
```

Comment:
Leaving out the naming: `"pnc-users-rex-admin", "pnc-users-rex-user", "pnc-users-admin"`